test/testenv/appframework_utils.go

```diff
@@ -8,6 +8,36 @@ import (
+	"Add-on-for-ldap_400.tgz":                            {"App-name": "TA-LDAP", "V1": "4.0.0", "V2": "4.0.0", "V2filename": "Add-on-for-ldap_400.tgz"},
+	"splunk-es-content-update_3200.tgz":                  {"App-name": "DA-ESS-ContentUpdate", "V1": "3.2.0.0", "V2": "3.2.1.0", "V2filename": "splunk-es-content-update_3210.tgz"},
+	"Palo-alto-networks-add-on-for-splunk_660.tgz":       {"App-name": "Splunk_TA_paloalto", "V1": "6.6.0", "V2": "7.0.0", "V2filename": "Palo-alto-networks-add-on-for-splunk_700.tgz"},
+	"microsoft-azure-add-on-for-splunk_300.tgz":          {"App-name": "TA-MS-AAD", "V1": "3.0.0", "V2": "3.1.1", "V2filename": "microsoft-azure-add-on-for-splunk_311.tgz"},
+	"splunk-add-on-for-unix-and-linux_830.tgz":           {"App-name": "Splunk_TA_nix", "V1": "8.3.0", "V2": "8.3.0", "V2filename": "splunk-add-on-for-unix-and-linux_830.tgz"},
+	"splunk-app-for-microsoft-exchange_401.tgz":          {"App-name": "splunk_app_microsoft_exchange", "V1": "4.0.1", "V2": "4.0.2", "V2filename": "splunk-app-for-microsoft-exchange_402.tgz"},
+	"Splunk-app-for-aws_601.tgz":                         {"App-name": "splunk_app_aws", "V1": "6.0.1", "V2": "6.0.2", "V2filename": "Splunk-app-for-aws_602.tgz"},
+	"splunk-machine-learning-toolkit_520.tgz":            {"App-name": "Splunk_ML_Toolkit", "V1": "5.2.0", "V2": "5.2.1", "V2filename": "splunk-machine-learning-toolkit_521.tgz"},
+	"splunk-add-on-for-microsoft-cloud-services_412.tgz": {"App-name": "Splunk_TA_microsoft-cloudservices", "V1": "4.1.2", "V2": "4.1.3", "V2filename": "splunk-add-on-for-microsoft-cloud-services_413.tgz"},
+	"Splunk-app-for-stream_720.tgz":                      {"App-name": "splunk_app_stream", "V1": "6.0.1", "V2": "6.0.2", "V2filename": "Splunk-app-for-stream_720.tgz"},
+	"Splunk-db-connect_350.tgz":                          {"App-name": "splunk_app_db_connect", "V1": "3.5.0", "V2": "3.5.1", "V2filename": "Splunk-db-connect_351.tgz"},
+	"Splunk-security-essentials_332.tgz":                 {"App-name": "Splunk_Security_Essentials", "V1": "3.3.2", "V2": "3.3.3", "V2filename": "Splunk-security-essentials_333.tgz"},
+	//Need to explore these apps for their app Name and installation steps
+	"splunk-app-for-vmware_401.tgz":          {"App-name": "", "V1": "4.0.1", "V2": "4.0.2", "V2filename": "splunk-app-for-vmware_402.tgz"},
+	"splunk-it-service-intelligence_472.spl": {"App-name": "", "V1": "4.7.2", "V2": "4.9.0", "V2filename": "splunk-it-service-intelligence_490.spl"},
+	"splunk-enterprise-security_640.spl":     {"App-name": "", "V1": "6.4.0", "V2": "6.4.1", "V2filename": "splunk-enterprise-security_641.spl"},
+}
+
+//BasicApps Apps that require no restart to be installed
+var BasicApps = []string{"splunk-common-information-model-cim_4181.tgz", "Add-on-for-ldap_400.tgz", "splunk-es-content-update_3200.tgz", "Palo-alto-networks-add-on-for-splunk_660.tgz", "microsoft-azure-add-on-for-splunk_300.tgz"}
```

Comment:
Is "Add-on-for-ldap_400.tgz" uploaded ?

Reply:
removed it. We can re add when we upload the app